bin/conpot

```diff
@@ -16,7 +16,7 @@
 # Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-import gevent.monkey; gevent.monkey.patch_all()
+from gevent import monkey; monkey.patch_all()
```

Comment:
How is this change different from what it was previously? :slightly_smiling_face:

Reply:
This way all monkey patching lines consistently use the same syntax. Before they differed from file to file.

Really it stems from my IDE preferring `from foo import bar` over `import foo.bar`.

It's a non-functional change so I can undo it if you want.